Index: Init.php
===================================================================
--- Init.php    (revision 320)
+++ Init.php    (working copy)
@@ -176,6 +176,12 @@
                                'siprop' => 'extensions|general',
                )));
                
+               if( isset( $siteinfo['error'] ) && $siteinfo['error']['code'] == 'readapidenied' ) {
+                       global $pgHooks;
+                       $pgHooks['PostLogin'][] = array( 'Peachy::wikiChecks', $base_url );
+                       return array( MINMW, array() );
+               }
+
                $version = preg_replace( '/[^0-9\.]/', '', $siteinfo['query']['general']['generator'] );
 
                if( version_compare( $version, MINMW ) < 0 ) {
Index: RELEASE-NOTES
===================================================================
--- RELEASE-NOTES       (revision 320)
+++ RELEASE-NOTES       (working copy)
@@ -9,10 +9,11 @@
 *
 
 === New features in 1.1 ===
-*(bug 64) Hooks added to HTTP class
+* (bug 64) Hooks added to HTTP class
+* 'colorized' option added to Diff function
 
 === Bug fixes in 1.1 ===
-*
+* (bug 63) Check if it is a read only wiki before running wikiChecks
 
 == Compatibility ==
 
Index: Includes/Hooks.php
===================================================================
--- Includes/Hooks.php  (revision 320)
+++ Includes/Hooks.php  (working copy)
@@ -71,7 +71,7 @@
                        }
 
                        if( isset( $data ) ) {
-                               $args += array($data);
+                               $args = array_merge( array($data), $args );
                        }                          
                         
                        if( isset( $object ) ) {
